taxonomies/food/ingredients.txt

```diff
@@ -37380,7 +37375,7 @@ et:jõhvikas, harilik jõhvikas
 fa:کرنبری
 fi:karpalo, karpalot
 fo:tranuber
-fr:airelles, cranberries, cranberry, cranberrys, canneberge
+fr:airelles, cranberries, cranberry, cranberrys, canneberge, Canneberge crue, Cranberry crue
```

Comment:
airelles and canneberges are different fruits

Reply:
@moon-rabbitOFF We currently mix "airelles" and "canneberges" before this PR, so I suggest we merge it as-is, and it would be great if you could create another PR that would make the distinction between airelles and canneberges. From reading https://fr.wikipedia.org/wiki/Vaccinium I'm completely lost..